lib/kubernetes-deploy/statsd.rb

```diff
@@ -4,39 +4,25 @@
 
 module KubernetesDeploy
   class StatsD
-    extend ::StatsD
-
     PREFIX = "KubernetesDeploy"
 
     def self.duration(start_time)
       (Time.now.utc - start_time).round(1)
     end
 
-    def self.build
-      if ENV['STATSD_DEV'].present?
-        self.backend = ::StatsD::Instrument::Backends::LoggerBackend.new(Logger.new($stderr))
-      elsif ENV['STATSD_ADDR'].present?
-        statsd_impl = ENV['STATSD_IMPLEMENTATION'].present? ? ENV['STATSD_IMPLEMENTATION'] : "datadog"
-        self.backend = ::StatsD::Instrument::Backends::UDPBackend.new(ENV['STATSD_ADDR'], statsd_impl)
-      else
-        self.backend = ::StatsD::Instrument::Backends::NullBackend.new
+    def self.client
+      @client ||= begin
+        sink = if ::StatsD::Instrument::Environment.current.env.fetch('STATSD_ENV', nil) == 'development'
```

Comment:
It seems as though the `::StatsD::Instrument::Environment.current.env` is just a [basic wrapper around `ENV`](https://github.com/Shopify/statsd-instrument/blob/master/lib/statsd/instrument/environment.rb#L8), in this case. The other fallbacks appear to be geared more towards Rails applications, of which krane isn't one. With that in mind, it might be cleaner just to do `if ENV.fetch(X, nil) == Y`, instead.